PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md

```diff
@@ -0,0 +1,65 @@
+#### Name
+
+Mac - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+REMOTE_DMG_URL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
+APP_NAME="Jumpcloud Assist App.app"
+LOCAL_DMG_PATH="/tmp/jumpcloud-assist-app.dmg"
+
+for SIGNAL in TERM KILL; do
+    PID=
+    for _ in $(seq 5); do
+        PID=$(launchctl list | grep com.jumpcloud.assist | cut -f1)
```

Comment:
This will not work as the app is not a deamon. The suggestion below will fix the issue. The line checks for the parent process ```Jumpcloud Assist App$``` and terminates it.
```suggestion
        PID=$(ps -ef | grep -iE "Jumpcloud Assist App$" | awk '{printf $2}')
```

Reply:
I don't know the rules for what makes a Mac App a "daemon" or not, but it does show up in the output of `launchctl list`

```
% launchctl list | grep -i com.jumpcloud.assist
6381	0	com.jumpcloud.assist-app.2276
```

Reply:
My macOS did not show your output. I am using Monterrey 12.0. Which version of macOS are you using? Can you test with my suggestion on your test environment?

Reply:
I'm not convinced the `ps` solution works. 
<img width="778" alt="image" src="https://user-images.githubusercontent.com/87679356/182715575-f18b9f22-f88a-439c-80af-397ffea3a032.png">

Reply:
Small typo, thanks